modules/fretboard/index.js

```diff
@@ -71,3 +71,50 @@ export function scale (tuning, scale, first, last) {
   var filter = map(scaleFilter(scale))
   return build(tuning, first, last).map(filter)
 }
+
+/**
+ * Build an array of reachable chord shapes based on given notes and tuning.
+ * @param {String|Array} tuning - the tuning name or notes
+ * @param {Array} notes - an array of chord notes
+ * @param {Integer} first - the first fret number
+ * @param {Integer} last - the last fret number
+ * @return {Array} An array of arrays, one for each possible shape.  Index is string index [ 'E2', 'B2', 'E3', 'G#3', 'B3', 'E4' ]
+ */
+export function chordShapes (tuning, notes, first, last, span) {
```

Comment:
I think we could find a better name than `chordShapes`. Something more concrete and descriptive. What about: chordPositions? I don't have the perfect name, but we should try to find it (tonal has a quite big API and it's important to grasp easily what is all about)